tests/integration/test_storage_rabbitmq/test_failed_connection.py

```diff
@@ -182,8 +185,8 @@ def test_rabbitmq_restore_failed_connection_without_losses_1(rabbitmq_cluster, r
         CREATE TABLE test.consume (key UInt64, value UInt64)
             ENGINE = RabbitMQ
             SETTINGS rabbitmq_host_port = 'rabbitmq1:5672',
-                    rabbitmq_flush_interval_ms=500,
-                    rabbitmq_max_block_size = 100,
+                    rabbitmq_flush_interval_ms=1000,
+                    rabbitmq_max_block_size = 1,
```

Comment:
It seems to me that it must lead to slow down instead 🤔

Reply:
Yes, it slows down throughput in general, but since I reduced so much the number of messages, the time it takes for the test to run is still better.

The thing here is that we need the ingestion to be slow enough so that we have time to suspend the RabbtiMQ server before ClickHouse consumes all messages, while still not taking forever to finish the test due to a huge amount of messages.

Reducing `rabbitmq_max_block_size` and the number of messages is the best way I found to do it, but I'm open to new ideas.